src/qt/sendcoinsdialog.cpp

```diff
@@ -329,8 +329,17 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
     question_string.append(tr("Do you want to create this transaction?"));
     question_string.append("<br /><span style='font-size:10pt;'>");
     if (model->wallet().privateKeysDisabled() && !model->wallet().hasExternalSigner()) {
+        /*: Text to inform a user attempting to create a transaction of their current options. At this stage,
```

Comment:
In commit "qt: hide Create Unsigned button behind an expert mode option" (790733e54c3daff547acbc90fb179b48b6d2fb3c)

Minor: This changes seem like they should be part of the previous commit. The previous commit is adding the "create unsigned" button unconditionally, and the new commit is adding the option to hide the button. It seems like the instructions for using the button should be added in the same commit that adds the button.

This is nitpicking though, so ignore unless there's some need to change.